docs/Personas.md

```diff
@@ -0,0 +1,9 @@
+### Personas
+
+- Eugenio Trujillo, cacereño, 35 años, programador y escalador aficionado con más de 15 años de
+  experiencia.
+- África Canto, madrileña, 28 años, bombera y escaladora poco experimentada.
```

Comment:
Yo aquí entiendo dos cosas
1. Bombera: muy buena forma física. Eso puede tener trascendencia o no, tenéis que plasmarlo en las HUs o más adelante
2. Poco experimentada. Entiendo que la experiencia tiene importancia, y que los problemas (HUs) van a depender de la experiencia

Reply:
Resuelto modificando historias de usuario